lib/install/stylesheets/tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  // Purge unused TailwindCSS styles
+  purge: {
+    enabled: ["production"].includes(process.env.NODE_ENV),
+    content: [
+      './**/*.html.erb',
+      './app/helpers/**/*.rb',
```

Comment:
```suggestion
      './app/views/**/*.*,
      './app/helpers/**/*.rb',
```

IMO this should match the compressor: https://github.com/rails/tailwindcss-rails/blob/main/lib/tailwindcss/compressor.rb#L14, so that either way you use this gem you get the same outcome.